client/packages/common/package.json

```diff
@@ -14,6 +14,11 @@
     "@capacitor/core": "^3.5.1",
     "@emotion/react": "^11.7.1",
     "@emotion/styled": "^11.6.0",
+    "@jsonforms/core": "^3.0.0-beta.3",
+    "@jsonforms/material-renderers": "^3.0.0-beta.3",
+    "@jsonforms/react": "^3.0.0-beta.3",
+    "@material-ui/icons": "^4.11.3",
+    "@mui/icons-material": "^5.8.0",
```

Comment:
I don't think we need the icons, I'll try and remove in next PR

Reply:
please! the icons packages is huge and we have deliberately created only the icons we need in order to avoid it